File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = Counter(nums)
        
        # output = []

        # for i in range(k):
        #     output.append(count.most_common()[i][0])

        return [num for num, freq in count.most_common(k)]
        # return output

File: Data Structures & Algorithms/count-primes/submission-2.java
class Solution {
    public int countPrimes(int n) {
        
        if (n == 0 || n == 1){
            return 0;
        }

        int primes = 0;

        int [] nums = new int[n];
        Arrays.fill(nums,1);

        nums[0] = 0;
        nums[1] = 0;

        for (int num = 2; num < n; num++){
            if(nums[num] != 0){
                primes++;
                for(long i = (long) num * num; i < n; i += num){
                    nums[(int)i] = 0;
                }
            }
        }

        return primes;
    }
}